configurations.yml

```diff
@@ -0,0 +1,19 @@
+logging:
+  log_file: "./logs/out_{time}.log"
+  format: "{time:YYYY-MM-DD at HH:mm:ss} {level} {message}"
+  backtrace: true
+  diagnose: true
+  level: "DEBUG"
+  retention: "7 days"
```

Comment:
Add this in as a comment:
"See https://loguru.readthedocs.io/en/stable/api/logger.html"

Reply:
Done